tasks.py

```diff
@@ -61,9 +61,6 @@ def create_env_file(env_file):
 
 # Project setup and update
 def l10n_block_inventory(ctx):
-    print("* Updating localizable fields")
-    l10n_sync(ctx)
-    l10n_update(ctx)
```

Comment:
These are related to the TODO in the comment above. We may need to add something like this back in at some point. I'm not 100% certain we'll need to, but since modeltranslation was removed we don't need these anymore.